content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -451,17 +449,7 @@ avoid unnecessary retries within an index.
 
 ### Pod failure policy {#pod-failure-policy}
 
-{{< feature-state for_k8s_version="v1.26" state="beta" >}}
-
-{{< note >}}
-You can only configure a Pod failure policy for a Job if you have the
-`JobPodFailurePolicy` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-enabled in your cluster. Additionally, it is recommended
-to enable the `PodDisruptionConditions` feature gate in order to be able to detect and handle
-Pod disruption conditions in the Pod failure policy (see also:
-[Pod disruption conditions](/docs/concepts/workloads/pods/disruptions#pod-disruption-conditions)).
-Both feature gates are available in Kubernetes {{< skew currentVersion >}}.
-{{< /note >}}
```

Comment:
I'm wondering if this note still would be valid since these feature gates still exist.
So, shouldn't we remove this note when we remove these feature gates?
But, I'm not sure when we should remove such a note. Is there any guideline for this situation?

Reply:
Yes, this is obsolete. Removed. PTAL

Reply:
> Is there any guideline for this situation?

I'm not sure about that, but it seems the practice, for example here: https://github.com/kubernetes/website/pull/46991/files, or here https://github.com/kubernetes/website/pull/45297/files

Reply:
Oh, I see. Thank you for letting me know!